### ISSUE TITLE
Calendar - Calendar Conflicting plugin commands & names

### ISSUE THREAD

**Issue by anoduck** [itchyny/calendar.vim#100]
I first wanted to thank you for writing a awesome plugin. It is by far the best calendar plugin/package/addon that I have found in the editor world. It makes life so much easier. 

Second, as you probably know it conflicts with matsumoto's calendar plugin. To reiterate the situation:

Both are named calendar.vim, and when inside of vim you call `:Calendar` either your plugin or matsumoto's plugin loads depending where it is located inside of your vimrc. Whichever plugin you are able to execute, the other you are not. Since org-mode for vim and journal.vim use matsumoto's plugin, it makes it impossible to have both enabled inside of your vimrc and/or use both at the same time. 

I have attempted to find information concerning creating a location specific key map, that is a keymap that refers to a location inside of the load path. ex.` /home/user/.vim/bundle/Itchyny'sAwesomeCalendarplugin/plugin/calendar.vim = <map>something.` But have been unable to find information about such a keymap, or how to do it. I have even searched the plugins for vim. I usually use emacs, but your plugin is sweet. I figured it would be easier to trouble you since your plugin is cleanly written and organized, and a modification or some form there of, might be a requirement. But, I could be wrong. 

Thanks,

**Comment by itchyny** [comment]
I should apologize on this.  We cannot use both my plugin and mattn's calendar plugin. This plugin may not be able to be used with a plugin which interact with mattn's plugin.
This problem was actually supposed to appear since I was developing the plugin. I actually renamed my plugin multiple times (neo-calendar, ultimate-calendar, zcalendar or something like that) but I couldn't be satisfied with writing such function prefix again and again (like `zcalendar#foo#bar`). So after all I decided to name this plugin just `calendar` and make it a *replacement* of mattn's calendar. Some people blamed me for creating a conflict intentionally, and claim that how the users choose plugins should be decided by the users. I agree with the opinion but I could not come up with a better command/function prefix than calendar; this plugin is actually a calendar in the editor too.
